Python/Product/VSInterpreters/PackageManager/PipRequirementsUtils.cs

```diff
@@ -91,7 +91,7 @@ IEnumerable<PackageSpec> installed
                 var installedPackage = installed.FirstOrDefault(pkg =>
                     string.Compare(pkg.Name, requirementPackageName, StringComparison.OrdinalIgnoreCase) == 0);
 
-                if (!requirementPackageName.Equals("") && installedPackage == null) {
+                if (!requirementPackageName.Equals("", StringComparison.InvariantCulture) && installedPackage == null) {
```

Comment:
`if (requirementPackageName.Length > 0 && installedPackage == null)`

Reply:
Modified in next commit